src/App.tsx

```
@@ -5,6 +5,15 @@ function App() {
   return (
     <div className='w-full h-full flex flex-col items-center justify-center'>
       <h1 className='text-5xl font-bold'>Deeeply</h1>
+      <button
+        type="button"
+        className='mt-4 bg-blue-500 text-white px-4 py-2 rounded'
+        onClick={() => {
+          throw new Error("Sentry Test Error");
+        }}
+      >
```

ESLint

[error] 'React' must be in scope when using JSX

(react/react-in-jsx-scope)